packages/webdriverio/src/index.js

```diff
@@ -1,3 +1,4 @@
+/// <reference types="webdriverio/webdriverio"/>
```

Comment:
this is a js file, no need for a ts reference statement

Reply:
I think it does, this will be interpreted by the package as global type definitions even though this is a JS file. To show this off I removed it and linter fails because it cannot find the namespace anymore. You can see the same error in the editor:
![image](https://user-images.githubusercontent.com/15076656/95691822-d95f6700-0c2a-11eb-97f6-1cc5bd6b77da.png)

My suggestion is that we keep this reference in this file.

Reply:
So @christian-bromann, what do you say, should we keep this reference here? 🙂 

I think it will make refactoring of the module easier, especially if these types are intended to be globally available in its scope. Otherwise, any suggestions?

Reply:
I guess we should. Still, weird that the TS compiler even notices this in a js file.

Reply:
Ok, found a more elegant solution: https://github.com/webdriverio/webdriverio/pull/5957/commits/dd0d8669c0802e66e509b859fe3b50c8b5c23553